sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/models/Documents.java

```diff
@@ -4,22 +4,37 @@
 
 import java.util.List;
 
-// A (paginated) list of documents from CosmosDB
+/* A (paginated) list of documents from CosmosDB. */
```

Comment:
`/**` for Javadoc

Reply:
Also please commit using Android Studio options to reformat:

<img width="275" alt="screen shot 2019-02-27 at 4 32 00 pm" src="https://user-images.githubusercontent.com/623672/53532814-38e81d80-3aad-11e9-8703-4f1106f772c2.png">

Multiple files are formatted incorrectly.

Reply:
wow did not know that!